### DISCUSSION TITLE
Im getting Error while connecting ganache with foundry

### DISCUSSION THREAD

**Discussion by Navin77773** [Cyfrin/foundry-full-course-cu#181] [Q&A] [answered]
command '''forge create SimpleStorage --rpc-url http://127.0.0.1:8545 --interactive

error
'''Compiling...
No files changed, compilation skipped
Error: 
error sending request for url (http://127.0.0.1:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)'''

**Comment by cromewar** [comment]
Hello @Navin77773 
We will need more context on this please.
Which OS you are using?
have you run ganache already running?

**Reply by Navin77773** [reply]
yes i have installed my ganache , and just changed the server to wsl , im using windows os (should i change anything otherthan that in ganache)

**Reply by PatrickAlphaC** [reply, marked as answer]
Just work with anvil and ignore the ganache part. I promise if you're running into issues here it's not worth the effort. Anvil is basically ganache without a flashy UI.
